translate/filters/checks.py

```diff
@@ -2155,10 +2161,31 @@ def accelerators(self, str1, str2):
         """Checks whether accelerators are consistent between the
         two strings.
 
-        For Mozilla we lower the severity to cosmetic.
+        For Mozilla we lower the severity to cosmetic, and for some languages
```

Comment:
With this change we could now bring back severe failure.  But lets do that outside of this PR